aspnetcore/tutorials/razor-pages/razor-pages-start.md

```diff
@@ -35,7 +35,11 @@ There are three versions of this tutorial:
 * Create a new ASP.NET Core Web Application. Name the project **RazorPagesMovie**. It's important to name the project *RazorPagesMovie* so the namespaces will match when you copy/paste code.
   ![new ASP.NET Core Web Application](../../mvc/razor-pages/index/_static/np.png)
 * Select **ASP.NET Core 2.0** in the dropdown, and then select **Web Application**.
-  ![Web Application (Razor Pages)](../../mvc/razor-pages/index/_static/np2.png)
+
+> [!NOTE]
+> If you want to use ASP.NET Core on the .NET Framework, you must first select **.NET Framework** from the leftmost dropdown in the dialog, then you can select the desired ASP.NET Core version.
```

Comment:
To use .NET Framework with ASP.NET Core, select **.NET Framework** from the leftmost dropdown in the dialog.

Reply:
Make this an include and add it everywhere. It doesn't matter that Web app is selected for the MVC apps. For bonus points you could have 2 images.